monocle/static/js/main.js

```diff
@@ -252,7 +261,7 @@ function addPokestopsToMap (data, map) {
         var icon = new PokestopIcon();
         var marker = L.marker([item.lat, item.lon], {icon: icon});
         marker.raw = item;
-        marker.bindPopup('<b>Pokestop ' + item.external_id + '</b>');
+		marker.bindPopup('<b>Pokestop: <p><p>Location : ' + item.lat + ',' + item.lon + '</p><p><a href=https://www.google.com/maps/?daddr='+ item.lat + ','+ item.lon +' target="_blank" title="See in Google Maps">Get directions</p></b>');	
```

Comment:
Hi @evenly-epic-mule thanks   for show  my mistakes, i try to learn more python and improve every time. 👍